itou/templates/apply/process_details_company.html

```diff
@@ -1,12 +1,38 @@
 {% extends "apply/process_base.html" %}
 {% load django_bootstrap5 %}
 {% load format_filters %}
+{% load theme_inclusion %}
 {% load matomo %}
 
 {% block actions %}
     {% include "apply/includes/siae_hiring_actions.html" with job_application=job_application %}
 {% endblock %}
 
+{% block content_title %}
+    {{ block.super }}
+
+    {# Temporary advertisement alert #}
+    <div class="alert alert-important alert-dismissible fade show" role="status">
+        <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Fermer"></button>
+        <div class="row">
+            <div class="d-none d-md-inline col-md-auto">
+                <img src="{% static_theme_images 'ico-bicro-important.svg' %}" alt="" height="80">
+            </div>
+            <div class="col-12 col-md px-md-0">
+                <p class="mb-2">
+                    <strong>Page PASS IAE</strong>
+                </p>
+                <p class="mb-0">
+                    Une page dédiée au PASS IAE est actuellement en construction. Vos retours sont importants, aidez-nous en réalisant ce test de 5 minutes
```

Comment:
```suggestion
                    Une page dédiée au PASS IAE est actuellement en construction. Vos retours sont importants, aidez-nous en réalisant ce test de 5 minutes.
```
Et pareil en dessous ?